src/framework/auth/services/token/token.ts

```diff
@@ -14,6 +14,14 @@ export abstract class NbAuthToken {
   }
 }
 
+export class InvalidJWTTokenError extends Error {
+  constructor(message: string) {
+    super(message);
+    const actualPrototype = new.target.prototype;
+    Object.setPrototypeOf(this, actualPrototype);
+  }
+}
+
```

Comment:
we also need another error then, as we cannot use `InvalidJWTTokenError` everywhere, since not every token is JWT token.
probably like this
`NbAuthInvalidTokenError`
`NbAuthInvalidJWTTokenError`